tests/integration/targets/test_mysql_query/tasks/mysql_query_initial.yml

```diff
@@ -289,6 +289,38 @@
       - result is failed
       - result.msg is search('the elements in query list must be strings')
 
+  - name: Create {{ test_table4 }}
+    mysql_query:
+      <<: *mysql_params
+      login_db: '{{ test_db }}'
+      query: 'CREATE TABLE {{ test_table4 }} (id int primary key, story text)'
+
+  - name: Insert test data using replace statement
+    mysql_query:
+      <<: *mysql_params
+      login_db: '{{ test_db }}'
+      query: "REPLACE INTO {{ test_table4 }} VALUES (1, 'first')"
+      single_transaction: yes
+    register: result
+
+  - assert:
+      that:
+      - result is changed
+      - result.rowcount == [1]
+
+  - name: Replace test data
+      mysql_query:
+        <<: *mysql_params
+        login_db: '{{ test_db }}'
+        query: "REPLACE INTO {{ test_table4 }} VALUES (1, 'one')"
+        single_transaction: yes
+      register: result
```

Comment:
```suggestion
    mysql_query:
      <<: *mysql_params
       ogin_db: '{{ test_db }}'
      query: "REPLACE INTO {{ test_table4 }} VALUES (1, 'one')"
      single_transaction: yes
    register: result
```

Reply:
Ah, indent issue, I just committed to fix that.